.github/workflows/captains-test.yml

```diff
@@ -4,7 +4,7 @@ on: [push]
 
 jobs:
   build:
-
+    name: Run Tests
```

Comment:
Don't know why this got in here, but it is fine. It will enable the other PR when this is merged.